docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -0,0 +1,59 @@
+
+[#_add_debug_logging_to_application_vertx]
+= Adding log output to your {VertX} application.
+
+. To add logging for your application, define a method to access the `Logger` class:
```

Comment:
I suggest giving the full path to the logger class here, e.g.:

> To add logging for your application, define a method to access the `com.x.y.Logger` class

Reply:
You can keep the code example as is.

Reply:
I don't think "define a method" is a proper wording here. A method isn't defined here (or declared), it's _called_. Also there's a colon at the end of this line, and the next line contains another sentence instead of an example.

In general, I think these two lines are a bit too verbose; I think we can safely assume that most Java developers are familiar with how their logging library of choice is used. I think these two lines could be replaced by

> To add logging to your application, define a `io.vertx.core.logging.Logger`:

This actually has one advantage over the existing text -- it explicitly says that it's the Vert.x logging facade that is being used here.